table.rst

```diff
@@ -0,0 +1,49 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              | pd.Int8Dtype,           |
+|                   | pd.Int16Dtype,          |
+|                   | pd.Int32Dtype,          |
+|                   | pd.Int54Dtype,          |
+|                   | "int",int,np.int8,      |
+|                   |  np.int16,np.int32,     |
+|                   |  np.int64,np.uint8      |
+|                   |  np.uint16,np.uint32    |
+|                   |  np.uint64,np.intp      |
+|                   |  np.uintp,np.integer    |
++-------------------+-------------------------+
+|  str              | StringDtype,str,"str"   |
++-------------------+-------------------------+
+|  bytes            | np.byte,np.ubyte,bytes  |
+|                   | "bytes"                 | 
++-------------------+-------------------------+
+|  float            | pd.Float32Dtype,        |
+|                   | pd.Float64Dtype,        |
+|                   | float,"float",np.float16|
+|                   | np.float32,np.float64   |
+|                   | np.float96,np.float128  |
+|                   | np.floating             |
++-------------------+-------------------------+
+|  complex          | "complex",np.complex64, |
+|                   | np.complex128,          |
+|                   | np.complex192,          |
+|                   | np.complex256,          |
+|                   | complex                 |
++-------------------+-------------------------+
+|  bool             | pd.BooleanDtype, bool   |
+|                   | np.bool_                |
+|                   | "bool"                  |
++-------------------+-------------------------+
+| Timedelta         | np.timedelta64,         |
+|                   | Timedelta,              |
+|                   | "timedelta64[ns]"       |
+|                   |                         |
++-------------------+-------------------------+
+|  Timestamp        | Timestamp               |
```

Comment:
can't pass `Timestamp` as a `Dtype`, so remove

Reply:
Done the changes does the table feel fine